docs/arcaflow/workflows/flow-control.md

```diff
@@ -107,3 +107,179 @@ outputs:
     my_plugin: !expr $.steps.my_plugin.outputs.success
     my_other_plugin: !expr $.steps.my_other_plugin.outputs.success
 ```
+
+### Dry Out with bindConstants()
+
+The builtin function [bindConstants()](expressions.md#functions) allows you to factor out any constant input variables to a foreach subworkflow.
```

Comment:
This lead-in I think suggests that what follows is an example. What immediately follows is in fact the "old style" equivalent that demonstrates the advantage of the new way ... so I'd suggest rephrasing this, or adding a second paragraph like "Here's the way you'd have to do this before `bindConstants`, repeating your constant for each iteration of the `for_each`:"